### DISCUSSION TITLE
Concatenated QR Code in C#

### DISCUSSION THREAD

**Discussion by rmrudro** [micjahn/ZXing.Net#348] [General]
Hello Everyone I want to know how I can implement Concatenated QR Code using Zxing . I work in a Japanease company . I implemented QR code using Zxing I can read and Write QR Code . But I have a question How can I implement Concatenated QR Code. First I want to say what is Concatenated QR Code is . 
[Supplemental Information on Concatenated QR Code]

Concatenated QR code is not a function to "divide a character string into multiple symbols".
It is a function to "express one character string with multiple symbols".

To read the symbol output as a concatenated QR code, a reader that supports the concatenation function is required.
Since each symbol also contains information such as "what number out of what", the symbol alone cannot read the correct character string.

If you set double-byte characters, single-character data may be output across two symbols. This is by design.

Many QR code reader apps that can be used on smartphones and tablets do not support the link function.
To check the output of the linked QR code, install an app that announces support for the linked function, or use a dedicated scanner.

Concatenated QR code is written 連結QRコード.  Here is the Ruby site It is also written what is Linked or Concatenated QR code.
You can also check it from here [https://linuxtut.com/ruby-generate-concatenated-qr-code-with-rqrcode-(knowledge)-3dd4c/](url)
Here it is written What is a linked QR code?
The QR code has a "concatenation function" that divides long data into up to 16 QR codes and combines them when reading. (For details, refer to QR Code.com)
In Japan, you will often see prescriptions from hospitals / clinics.
The output function is supported by third-party form output libraries, but it is difficult to find it in the OSS library, so I have been using it rqrcode. I modified rqrcode) so that it can be output.
This time, the target is to output the concatenated QR code to be recorded in the above "prescription", so only the QR code in 8bit_byte mode is targeted.
In this Japanease site there is also implemented Concatenated QR Code 
[https://programresource.net/2013/05/04/2193.html](url)
But is written in Java Can you help me how to implement Concatenated QR Code

**Comment by micjahn** [comment]
Please take a look at my answer to issue #349.
